opencog/atoms/bind/DefineLink.cc

```diff
@@ -84,4 +78,23 @@ DefineLink::DefineLink(Link &l)
 	init(l.getOutgoingSet());
 }
 
+Handle DefineLink::get_definition(const Handle& alias) {
+	// Get all DefineLinks associated with that alias, beware that it
+	// will also return DefineLink with that alias as definition body.
+    IncomingSet defs = alias->getIncomingSetByType(DEFINE_LINK);
+
+    // Return the first (supposedly unique) definition
+    for (LinkPtr defl : defs) {
+	    DefineLinkPtr def(DefineLinkCast(defl->getHandle()));
+	    if (def->get_alias() == alias)
```

Comment:
yeah, but how could that ever happen?    There's no way to construct the define without the definition, and there is no way to erase the definition without also erasing the define link.  Right?